tests/static/saveToCase/close_property.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>Untitled Form</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/E1B03643-109D-4DC6-9737-815E4401DE79" uiVersion="1" version="1" name="Untitled Form">
+					<name />
+					<save_to_case vellum:role="SaveToCase">
+						<c:case xmlns:c="http://commcarehq.org/case/transaction/v2" case_id="" date_modified="" user_id="">
```

Comment:
I was using an example from a skype convo with @ctsims 

> [2:35:50 PM] Clayton Sims: One caveat:
[2:35:57 PM] Clayton Sims: The block should be nested in the question id
[2:36:03 PM] Clayton Sims: so if the qid is "test_a"
[2:36:21 PM] Clayton Sims: the output should be 
 \<test_a\>
    <c:case xmlns:c="...
[2:36:28 PM] Clayton Sims: so we can keep things namespaced more or less

So it seems to be required